test/fixtures/valid-geojson.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
I see this fixture has 8 features and the last feature has a unique key of `Capital`. That is great because it will confirm the `num_features_to_query` option is working. However, I think it would be great to simplify the file so that it is easier to read. Ideally each features  could have different properties. While this is not very realistic it will be more useful as a "stress test" for this feature and more clearly state the intent of ensuring all feature properties are collected even if they are different.

Also, the fixture does not need an `id` property, that serves no purpose.

Also, it would be great to indent the JSON for readability (aka pretty print the file). For fun here is a crazy "one-liner" to do that:

```sh
node -e "var data=require('fs').readFileSync('test/fixtures/valid-geojson.json');require('fs').writeFileSync('test/fixtures/valid-geojson.json',JSON.stringify(JSON.parse(data),null,2))"
```